@xen-orchestra/web-core/lib/components/ui/tag/UiTagsList.vue

```diff
@@ -0,0 +1,14 @@
+<!-- v1 -->
+<template>
+  <div class="ui-tags-list">
+    <slot />
+  </div>
+</template>
+
+<style lang="postcss" scoped>
+.ui-tags-list {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Is flex-wrap necessary here ? Won't this potentially break the design? I'm not sure where this is going to be used